app/views/candidate_interface/contact_details/address/edit.html.erb

```diff
@@ -1,8 +1,8 @@
 <% content_for :title, title_with_error_prefix(t('page_titles.address'), @contact_details_form.errors.any?) %>
 <% if FeatureFlag.active?(:international_addresses) %>
-  <% content_for :before_content, govuk_back_link_to(candidate_interface_contact_details_edit_address_type_path) %>
+  <% content_for :before_content, govuk_back_link_to(:back) %>
```

Comment:
`:back` uses the referrer and where it's not available, it uses JavaScript to go back in the user's history: https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

The design system has some guidance regarding this (emphasis added) https://design-system.service.gov.uk/components/back-link/:

> Make sure the link takes users to the previous page they were on, in the state they last saw it. Where possible, **ensure it works even when JavaScript is not available.**
> 
> If this is not possible, **you should hide the back link when JavaScript is not available.**

So... we can't use `:back` in this way, because it doesn't work without JavaScript and where the browser history queue is empty (e.g. you open the page in a new window/tab, the button doesn't do anything).